Disable swap encryption by default, as it simply hangs in Debian
Stretch. Hopefully this is only temporary...

Index: ltsp/client/share/ltsp/ltsp_config.d/35encrypted-swap
===================================================================
--- ltsp.orig/client/share/ltsp/ltsp_config.d/35encrypted-swap
+++ /dev/null
@@ -1,4 +0,0 @@
-# enable encrypted swap when cryptsetup is installed
-if [ -z "$ENCRYPT_SWAP" ] && [ -x /sbin/cryptsetup ]; then
-    set_lts_var ENCRYPT_SWAP True
-fi
